File: lp2/PokemonV-C#/PokemonV/Item.cs
namespace PokemonV
{
    public class Item
    {
        public Item(string name, string description)
        {
            this.name = name;
            this.description = description;
        }

        protected string name;
        public string Name
        {
            get { return name; }
            set { name = value; }
        }

        protected string description;
        public string Description
        {
            get { return description; }
            set { description = value; }
        }
    }
}

File: lp2/PokemonV-Client/src/com/grupox/pokemonv/model/Item.java
package com.grupox.pokemonv.model;

public abstract class Item {
    /* Attributes */
    protected String name;
    protected int quantity;
    
    /* Constructor */
    public Item(int quantity){
        this.quantity = quantity;
    }
    
    /* Getters && Setters */
    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }

    public int getQuantity() {
        return quantity;
    }
    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }
}
